### ISSUE TITLE
App doesn't really delete files

### ISSUE THREAD

**Issue by jonaspacheco** [afollestad/cabinet-issue-tracker#212]
Everytime, even after quitting e reopening, restarting the phone, when I delete files they seemed to be gone, but when I plug the device to the computer and browse through the file explorer they are still there and need to be deleted again.

**Comment by afollestad** [comment]
I'm assuming you're talking about root, root is being redesigned at the moment.

**Comment by jonaspacheco** [comment]
it doesn't delete files from my downloads folder. is this a root routine?
if it is, OK. thanks for the quick reply.
 Em 07/05/2015 17:22, "Aidan Follestad" <notifications@github.com> escreveu:

**Comment by afollestad** [comment]
If it's in your internal storage, there should be no issue. It works fine
for me and no one else has reported any issues.

If you're talking about an SD Card or files not in internal storage then
it's most likely related to root access.

**Comment by jonaspacheco** [comment]
sorry, i don't know if i can help this far. i'm new to Android.

what i know... it's the internal storage. it's a moto x 2014. it doesn't
have external storage option.

I didn't rooted the phone. what happened was inside the default downloads
folder. I deleted the files and when I plugged the device on my to transfer
some music, the files were still there.

I can take screenshots or something to try to help.
Em 07/05/2015 22:25, "Aidan Follestad" <notifications@github.com> escreveu:
